sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -94,6 +94,11 @@ private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPip
                 {
                     activity.AddTag("otel.status_code", "ERROR");
                 }
+                else
+                {
+                    // Set the status to UNSET so the AppInsights doesn't try to infer it from the status code
+                    activity.AddTag("otel.status_code", "UNSET");
```

Comment:
is `otel.status_code` supported by appInsights SDK to populate success? It looks like [only EventHub listener is aware of it](https://github.com/microsoft/ApplicationInsights-dotnet/search?q=otel.status_code&type=)

Reply:
the listener you linked is used for all azure SDK activities including HTTP.